src/App.tsx

```diff
@@ -152,11 +163,36 @@ class Board extends React.Component<BoardProps, BoardState> {
         this.setState(newState);
     }
 
-    public handlePress = (e: KeyboardEvent) => {
-        let i = parseInt(e.key, 10);
-        console.log(i);
+    /**
+     * deselect / unhighlight all highlighted squares
+     */
+    public deselectSquares(): void {
+        const newState = _.cloneDeep(this.state);
+        newState.highlighting = false;
+        newState.highlights = new Set<SquareAddress>();
+        this.setState(newState);
+    }
+
+    public handleGlobalKeyDown = (e: KeyboardEvent) => {
+        if (e.keyCode === 8) { // Delete Key
+            e.preventDefault();
```

Comment:
Why are you calling preventDefault on keyboard events? Seems like you can allow them to bubble right?

Reply:
I call it on tab cause then it keeps cycling through all the clickable things on the screen. So after that, I just added it to everything else.